frb_example/with_flutter/lib/bridge_generated.io.dart

```diff
@@ -378,6 +397,19 @@ class FlutterRustBridgeExampleWire implements FlutterRustBridgeWireBase {
       _lookup<ffi.NativeFunction<ffi.Pointer<wire_uint_8_list> Function(ffi.Int32)>>('new_uint_8_list_0');
   late final _new_uint_8_list_0 = _new_uint_8_list_0Ptr.asFunction<ffi.Pointer<wire_uint_8_list> Function(int)>();
 
+  void drop_DartOpaqueFlutterRustBridgeExample(
```

Comment:
Is it something that is the same for all classes? If so, maybe we do not need to put it to *generated* code, but instead be in frb_dart/frb_rust

Reply:
fixed